pkg/sql/ttl/ttljob/ttljob_processor.go

```diff
@@ -300,14 +291,19 @@ func runTTLOnRange(
 
 	for {
 		// Check the job is enabled on every iteration.
-		if err := checkEnabled(&settings.SV); err != nil {
+		if err := checkEnabled(&serverCfg.Settings.SV); err != nil {
 			return rangeRowCount, err
 		}
 
 		// Step 1. Fetch some rows we want to delete using a historical
 		// SELECT query.
 		start := timeutil.Now()
 		expiredRowsPKs, err := selectBuilder.run(ctx, ie)
+		log.Dev.Infof(
```

Comment:
It looks like the nodes can select outside of the range they are leaseholders for so I need to fix that, but I'll do it separately.